boskos/metrics/metrics.go

```diff
@@ -39,32 +40,38 @@ var (
 	promMetrics = prometheusMetrics{
 		BoskosState: map[string]map[string]prometheus.Gauge{},
 	}
-	resources common.ResTypes
+	resources, states common.ResTypes
+	defaultStates     = []string{
+		common.Busy,
+		common.Cleaning,
+		common.Dirty,
+		common.Free,
+		common.Leased,
+	}
 )
 
 func init() {
 	flag.Var(&resources, "resource-type", "comma-separated list of resources need to have metrics collected")
+	flag.Var(&states, "resource-state", "comma-separated list of states need to have metrics collected")
```

Comment:
Would resource-states be a better name here (given that it is one flag)? Or is consistency with resource-type's lack of pluralization more important?